basics/data-import/pinot-input-formats.md

```diff
@@ -46,37 +46,39 @@ configs:
   multiValueDelimiter: '-'
 ```
 
-CSV Record Reader supports the following configs -
+CSV Record Reader supports the following configs:
 
-`fileFormat` - can be one of default, rfc4180, excel, tdf, mysql
+* **`fileFormat`**: Can be one of: default, rfc4180, excel, tdf, mysql.
 
-`header` - header of the file. The columnNames should be seperated by the delimiter mentioned in the config
+* **`header`**: Header of the file. The columnNames should be seperated by the delimiter mentioned in the configuration.
 
-`delimiter` - The character seperating the columns
+* **`delimiter`**: The character seperating the columns.
 
-`multiValueDelimiter` - The character seperating multiple values in a single column. This can be used to split a column into a list.
+* **`multiValueDelimiter`**: The character seperating multiple values in a single column. This can be used to split a column into a list.
```

Comment:
```suggestion
* **`multiValueDelimiter`**: The character separating multiple values in a single column. This can be used to split a column into a list.
```

Reply:
lets run spell check as part of the first pass to be sure